specification/advisor/resource-manager/Microsoft.Advisor/preview/2016-07-12-preview/advisor.json

```
@@ -398,6 +398,72 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/advisorScore": {
+      "get": {
+        "tags": [
+          "AdvisorScore"
+        ],
+        "description": "Retrieves data about the advisor score of the subscription",
+        "operationId": "AdvisorScore_List",
```

Usually operationId should be in pattern of `PuralNoun_Verb`, therefore
```suggestion
        "operationId": "AdvisorScores_List",
```